user_guide_src/source/general/configuration.rst

```diff
@@ -88,7 +88,9 @@ Environment Variables and CodeIgniter
 CodeIgniter makes it simple and painless to set Environment Variables by using a “dotenv” file. The term comes from the file name, which starts with a dot before the text “env”.
 
 CodeIgniter expects **.env** to be at the root of your project alongside the ``system``
-and ``app`` directories. There is a template file distributed with CodeIgniter that’s
+and ``app`` directories and you is able to change the name of .env file if you need manipulate
+a multiples applications connected with a single core system but with different environments at 
+``app/Config/Constants.php``. There is a template file distributed with CodeIgniter that’s 
 located at the project root named **env** (Notice there’s no dot (**.**) at the start?).
```

Comment:
```suggestion
and ``app`` directories. If you want to handle multiple environments in one installation you can change
the default file name via the **ENV_FILE** constant located in ``app/Config/Constants.php``. There is a template
file distributed with CodeIgniter that’s located at the project root named **env** (Notice there’s no dot (**.**) at the start?).
```

Reply:
> I think this is a good addition but we need some small changes and also the `env` file has to be restored.

Yes! Thank you. It's my first contribuiton to the CI4. I will follow your code types from now.